README.md

```diff
@@ -30,3 +30,38 @@ make test
 ```
 
 If you use VS Code, you should be sure to have installed the recommended extensions (Python, Ruff, and MyPy). Note that you'll be prompted to install these when you open the project in VS Code.
+
```

Comment:
To setup doc development you actually need to do a bit more (and we can rely on installing Quarto from pypi). This is the recipe (from the repo main dir):

```python
pip install -e ".[doc]"
```

We should also suggest that people install the Quarto VS Code extension.